<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
    <item>
     <widget class="QGraphicsView" name="Graphics">
      <property name="minimumSize">
       <size>
        <width>900</width>
        <height>900</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>900</width>
        <height>900</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>900</width>
        <height>900</height>
       </size>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="ActionsWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>135</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="MoveInputBox">
         <property name="title">
          <string>Переместить</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="dXLabel">
            <property name="text">
             <string>dX</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="dYLabel">
            <property name="text">
             <string>dY</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="dZLabel">
            <property name="text">
             <string>dZ</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QDoubleSpinBox" name="dXInput">
            <property name="minimum">
             <double>-2000000000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>2000000000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="dYInput">
            <property name="minimum">
             <double>-2000000000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>2000000000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QDoubleSpinBox" name="dZInput">
            <property name="minimum">
             <double>-2000000000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>2000000000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="3">
           <widget class="QPushButton" name="MoveButton">
            <property name="text">
             <string>Применить</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>134</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="RotateInputBox">
         <property name="title">
          <string>Поворот</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0">
          <item row="0" column="0">
           <widget class="QLabel" name="oXLabel">
            <property name="text">
             <string>θX</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="oYLabel">
            <property name="text">
             <string>θY</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="oZLabel">
            <property name="text">
             <string>θZ</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QDoubleSpinBox" name="oXInput">
            <property name="minimum">
             <double>-2000000000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>2000000000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="oYInput">
            <property name="minimum">
             <double>-2000000000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>2000000000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QDoubleSpinBox" name="oZInput">
            <property name="minimum">
             <double>-2000000000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>2000000000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="3">
           <widget class="QPushButton" name="RotateButton">
            <property name="text">
             <string>Применить</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>135</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="ScaleInputBox">
         <property name="title">
          <string>Масштабирование</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="kXLabel">
            <property name="text">
             <string>kX</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="kYLabel">
            <property name="text">
             <string>kY</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="kZLabel">
            <property name="text">
             <string>kZ</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QDoubleSpinBox" name="kXInput">
            <property name="minimum">
             <double>-2000000006.000000000000000</double>
            </property>
            <property name="maximum">
             <double>2000000000.000000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="kYInput">
            <property name="minimum">
             <double>-2000000000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>2000000000.000000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QDoubleSpinBox" name="kZInput">
            <property name="minimum">
             <double>-2000000000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>2000000000.000000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="3">
           <widget class="QPushButton" name="ScaleButton">
            <property name="text">
             <string>Применить</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>134</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="LoadButton">
         <property name="text">
          <string>Загрузить фигуру из файла</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="UnloadButton">
         <property name="text">
          <string>Загрузить фигуру в файл</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>135</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
